act-rules-format.bs

```diff
@@ -83,58 +83,43 @@ Even within a rules group, certain rules may not be used. This leaves one fewer
 
-The following test subject types are common in accessibility testing:
-
-- HTTP Response
-- DOM Tree
-- Rendered Page
-- Template
-- Script
-
-Other types of test subjects may be possible. In those cases the ACT Rule MUST include a detailed description of the test subject type, or a reference to the that description.
+Each ACT Rule MUST include a description of all the aspects under test by the rule. Each aspect MUST be discrete and no overlap exist between aspects. Some aspects are already well defined within the context of web content, such as HTTP messages, DOM tree, and CSS styling, and do not warrant a detailed description. Other aspects may not be well defined or even specific to web content. In these cases, an ACT Rule SHOULD include a detailed description, or a reference to one, of the aspect in question.
 
+Common Aspects {#input-aspects-common}
+--------------------------------------
 
-HTTP Response Testing {#input-types-http-response}
---------------------------------------------
+### HTTP Messages ### {#input-aspects-http}
 
-Testing the files as they are served to the web browser (or other user agent) has its limitations. The files may be manipulated in different ways through presentation and scripting. Although this is an excellent place for parser testing.
+The HTTP messages exchanged between a client and a server as part of requesting a web page may be of interest to ACT Rules that wish to perform validation of e.g. HTTP headers or unparsed HTML and CSS.
 
+### DOM Tree ### {#input-aspects-dom}
 
-DOM Tree Testing {#input-types-dom-tree}
------------------------------------
+The DOM tree constructed from parsing HTML, and optionally executing DOM manipulating JavaScript, may be of interest to ACT Rules that wish to test the structure of web pages. In the DOM tree, information about individual elements of a web page, and their relations, becomes available.
 
-After the web browser (or other user agent) has parsed the files, and executed scripts to get it into a specific state (be it the initial state or any other), tests can be run against the DOM Tree. The DOM Tree can be tested for things like correct parent-child relationships, use of required attributes or properties and more.
+The means by which the DOM tree is constructed, be it by a web browser or not, is not of importance as long as the construction follows any applicable specifications that might exist, such as [[DOM]].
 
+### CSS Styling ### {#input-aspects-css}
 
-Rendered Page Testing {#input-types-rendered-page}
---------------------------------------------
+The computed CSS styling resulting from parsing CSS and applying it to the DOM may be of interest to ACT Rules that wish to test the web page as presented to the user. Through CSS styling, information about the position, the fore- and background colors, the visibility, and more, of elements becomes available.
```

Comment:
Again, I would remove "wish to" and use the full word "foreground" rather than "fore-".